pom.xml

```diff
@@ -16,6 +16,7 @@
         <guava.version>16.0.1</guava.version>
         <slf4j.version>1.7.7</slf4j.version>
         <junit.version>4.9</junit.version>
+        <mockito.version>2.0.52-beta</mockito.version>
```

Comment:
It's the latest version from the Maven release repository. Even if has "beta" in its name, is a release. We can go back to the previous version without "beta", I think is 1.x.